### ISSUE TITLE
no VS2019 support?

### ISSUE THREAD

**Issue by RPG3D** [xmake-io/xmake#361]

**Comment by waruqi** [comment]
Yes, I haven't supported it yet. I will try to support it next week, or you can submit a pr to support it.

**Comment by waruqi** [comment]
I have supported vs2019 preview. You can install xmake from dev branch and try it.

**Comment by RPG3D** [comment]
Any doc for compile xmake?

**Comment by waruqi** [comment]
You can try https://github.com/xmake-io/xmake/releases/download/v2.2.3/xmake-dev.exe
